releases/release-1.28/release-notes/maps/pr-118329-map.yaml

```diff
@@ -0,0 +1,3 @@
+pr: 118329
+releasenote:
+  text: Fixed the alpha `CloudDualStackNodeIPs` feature.
```

Comment:
Minor note but we should probably enforce some sort of style consistency for a trailing period. @sanchita-07 do you know if there's a preference?

Reply:
Yes, we need to maintain style consistency in order to have a better reading experience. We can either follow the style of the previous version's release notes or maintain the style of the major release notes in the current version.